WalletWasabi.Backend/wwwroot/index.html

```diff
@@ -145,22 +145,38 @@ <h2 class="title">Non-custodial</h2>
                     <div class="col-md-6 offset-md-1">
                         <ul class="feature-list">
                             <li>
-                                <h2 class="title">Privacy is for everyone</h2>
-                                <p>After years of extensive research, we found a way to provide reliable onchain privacy even with unequal amount coinjoins. We named this wizardry, WabiSabi, meaning that we accept the inherent imperfections of Bitcoin. By providing arbitrary amount coinjoins, we are finally able to make bitcoin privacy more efficient and accessible for all.</p>
+                                <h2 class="title">Easy To Use</h2>
+                                <p>Wasabi is designed to be a <a href="https://www.researchgate.net/publication/228348285_Anonymity_loves_company_Usability_and_the_network_effect" target="_blank">user friendly</a> Bitcoin wallet, that handles users privacy automatically under the hood, including network connections, <a href="https://en.bitcoin.it/wiki/Transaction#Input" target="_blank">input</a> selection and coinjoining.</p>
                             </li>
                             <li>
-                                <h2 class="title">Easy to use</h2>
-                                <p>Wasabi is designed to be a user friendly Bitcoin wallet that manages a user's privacy automatically under the hood. Users dont need to worry about network connections, coin selection, coinjoining or change avoidance enabling even Peter McCormack to have good bitcoin privacy.</p>
+                                <h2 class="title">Comprehensive</h2>
```

Comment:
We should come up with a better title here.

Reply:
Yeah. Effortless?

Reply:
First I wanted to write inclusive but that didnt feel right either. I think the title should express how financial privacy is now possible for more people than ever before.